x/stake/keeper/delegation.go

```diff
@@ -430,7 +430,6 @@ func (k Keeper) unbond(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValA
 		k.RemoveValidator(ctx, validator.OperatorAddr)
 	}
 
-	k.OnDelegationSharesModified(ctx, delegation.DelegatorAddr, validator.OperatorAddr)
```

Comment:
Why are we no longer calling this here?

Reply:
it was a duplicate of being called at the top of the function!